frontend/src/api/reactions.js

```diff
@@ -11,7 +11,22 @@ const formatGeneRule = (reaction) => {
     if (w.match(/and|or/)) {
       return w;
     }
-    const gene = genes.find(g => g.id === w);
+
```

Comment:
As I commented in https://github.com/MetabolicAtlas/neo4j-data-generation/pull/18, due to the exists of multiple layers of parentheses, the parser needs to be able to handle such cases as well.

Reply:
[A fix](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/625/commits/f2f3933398a74e0623605526b9c72cc1a8c597b2) has been added to address this now. After you get this update as well as [the neo4j-data-generation one](https://github.com/MetabolicAtlas/neo4j-data-generation/pull/18/commits/c23570dadacd9341804a47630a8b9a15109ce859), you should be able to rebuild and test it on reactions with nested parenthesis. For example: `/explore/Human-GEM/gem-browser/reaction/ATPasel` should display the following.

<img width="1251" alt="image" src="https://user-images.githubusercontent.com/423498/122814282-422bd280-d2d4-11eb-93e6-d22ec18978ea.png">

Reply:
Btw, I do think there is a good refactoring opportunity to handle the gene rules but since it's a relatively serious bug I wanted to get this fix out as soon as possible.